public/js/bundle.js

```diff
@@ -93,6 +93,7 @@ class TableView {
 		this.headerRowEl = document.querySelector('THEAD TR');
```

Comment:
Your bundle.js file should not be checked into git or GitHub, since it's a big ugly mess of a file that production environments (Heroku) will generate for you anyway. This is why we put it in the .gitignore file at the beginning of development.

Delete from GitHub and don't commit any modifications to bundle.js in git.